src/client/applications/turnilo-application/turnilo-application.tsx

```diff
@@ -175,8 +176,11 @@ export class TurniloApplication extends React.Component<TurniloApplicationProps,
     const { view, timekeeper } = this.state;
 
     switch (view.viewType) {
-      case "oauth-message":
-        return <OauthMessageView oauth={appSettings.oauth}/>;
+      case "oauth-message": {
+        const oauth = appSettings.oauth;
+        if (!isEnabled(oauth)) throw new Error("OAuth configuration error");
```

Comment:
As a OOP I would do oauth.isEnabled :)
Make the error message more specific.

Reply:
This `if` is really to silence compiler here. I can't express in our types invariant that we never redirect to `oauth-message` if we don't have oauth enabled. 

If Turnilo would throw sometime this error that would mean something that we can't predict right now is really broken.

Reply:
> As a OOP I would do oauth.isEnabled :)

Best I can do is rename import :)